pkg/master/master.go

```diff
@@ -593,8 +594,8 @@ func (m *Master) init(c *Config) {
 		}
 		expAPIVersions := []api.GroupVersion{
 			{
-				GroupVersion: g.Group + "/" + expVersion.Version,
-				Version:      expVersion.Version,
+				GroupVersion: expVersion.Version,
```

Comment:
Rename the expVersion variable.

This is a good example of why it's really bad to have version sometimes mean "group and version".

Please add a test that would have caught this.